content/blog/2021-08-11-unconference.md

```diff
@@ -0,0 +1,21 @@
++++
+title="Unconference 2021"
+[extra]
+author="Nordic-RSE"
++++
+
+# Unconference 2021
+After our plans to meet the community in person in beautiful KTH in Stockholm were prevented by the world situation, we did not want to not meet the community this year, but at the same time were also busy with other work. In order to keep organizational hassle low, but still give the community a place to meet, learn and discuss we decided for the unconference/birds of a feather format. In that way the community could decide on the topics of interest and create an event to their liking.
```

Comment:
```suggestion
Although the world situation prevented our plans to meet the community in person in Stockholm and we were all busy with other works, we did not want to give up meeting the community this year. Thus we decided for the unconference/birds of a feather format to keep organizational hassle low, but still give the community a place to meet, learn, and discuss. In that way, the community could decide on the topics of interest and create an event to their liking.
```

Reply:
- "in person in beautiful KTH in Stockholm" have 3 "in"-s one after another, so it would be probably fine to say only Stockholm?
- "we did not want to not meet the community" : use of double not makes it a bit difficult to read, at least for me.
- The 2nd sentence: It could be personal preference, but I think having a conclusion first makes it easier to read.